
# message to set pid limits

uint8 dof

float32 output_min
float32 output_max
float32 integral_min
float32 integral_max


